data/json/monster_special_attacks/spells.json

```diff
@@ -138,13 +139,14 @@
     "name": { "str": "Spawn Raptors" },
     "description": "Summons many zombiespawn raptors.",
     "flags": [ "HOSTILE_SUMMON", "RANDOM_DAMAGE", "PERMANENT" ],
-    "valid_targets": [ "ground", "hostile" ],
-    "min_damage": 6,
-    "max_damage": 8,
-    "min_range": 3,
-    "max_range": 3,
-    "min_aoe": 4,
-    "max_aoe": 5,
+    "valid_targets": [ "ground", "self" ],
+    "hit_self": true,
+    "min_damage": 3,
+    "max_damage": 6,
+    "min_range": 0,
+    "max_range": 1,
```

Comment:
```suggestion
```
this spell has no level, so 0 is the max range this spell will be. 0 is the default, so no reason to bloat the json